sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -129,13 +130,12 @@ internal class RadarLocationManager(
     }
 
     fun stopTracking() {
-        RadarState.setStarted(context, false)
+        this.started = false
```

Comment:
No Android vitals difference when putting this in `SharedPreferences`, so can move back to memory